proto/bmeg/vest.proto

```diff
@@ -0,0 +1,25 @@
+
+// vest score
+
+syntax = "proto3";
+
+package bmeg;
+
+message VestScore {
```

Comment:
Perhaps it's better to put this in a `vest` namespace (e.g. `package vest;`) and drop the `Vest` prefix here?

Reply:
Following the pattern, all the other files in the bmeg dir use `package bmeg`.  Don't know.  Perhaps `package bmeg.vest`